Jenkinsfile

```diff
@@ -8,15 +8,25 @@ podTemplate(
             image: 'omisegoimages/blockchain-base:1.6-otp20',
             args: '${computer.jnlpmac} ${computer.name}',
             alwaysPullImage: true,
-            resourceRequestCpu: '1750m',
+            resourceRequestCpu: '500m',
```

Comment:
I changed only the `request` resources because without this changed there were no resources left for Postgres and whole Jenkins build didn't want to start